equinix/resource_network_ssh_key.go

```diff
@@ -16,17 +16,19 @@ import (
 )
 
 var networkSSHKeySchemaNames = map[string]string{
-	"UUID":  "uuid",
-	"Name":  "name",
-	"Value": "public_key",
-	"Type":  "type",
+	"UUID":      "uuid",
+	"Name":      "name",
+	"Value":     "public_key",
+	"Type":      "type",
+	"ProjectID": "project_id",
 }
 
 var networkSSHKeyDescriptions = map[string]string{
-	"UUID":  "The unique identifier of the key",
-	"Name":  "The name of SSH key used for identification",
-	"Value": "The SSH public key. If this is a file, it can be read using the file interpolation function",
-	"Type":  "The type of SSH key: RSA (default) or DSA",
+	"UUID":      "The unique identifier of the key",
+	"Name":      "The name of SSH key used for identification",
+	"Value":     "The SSH public key. If this is a file, it can be read using the file interpolation function",
+	"Type":      "The type of SSH key: RSA (default) or DSA",
+	"ProjectID": "The unique identifier of the resource-ssh public key",
```

Comment:
Same here. It can be changed to 'Unique identifier of the project that ssh public key is scoped to'.

Reply:
done